docs/workshop/part-ii/01-writing-your-first-promise.mdx

```diff
@@ -270,43 +265,71 @@ In a nutshell, a Kratix `Pipeline` is a series of containers that will be execut
+We will once again use the Kratix CLI to create the necessary files. Run the following command to bootstrap the skeleton of your Workflow:
 
-```bash
-mkdir -p workflows
-touch workflows/Dockerfile
+```
+kratix add container resource/configure/mypipeline --image kratix-workshop/app-pipeline-image:v1.0.0
 ```
 
-Open the `Dockerfile` in your editor and add the following content:
+The command above should have created a directory structure that looks like the following:
 
-```mdx-code-block
-import Dockerfile from "!!raw-loader!./_partials/writing-a-promise/03-Dockerfile"
+```
+.
+├── README.md
+├── dependencies
+│   └── dependencies.yaml
+├── example-resource.yaml
+├── promise.yaml
+└── workflows
+    └── resource
+        └── configure
+            └── mypipeline
+                └── kratix-workshop-app-pipeline-image
+                    ├── Dockerfile
+                    ├── resources
+                    └── scripts
+                        └── pipeline.sh
 ```
 
-<CodeBlock language="dockerfile" title="app-promise/workflows/Dockerfile">
-    {Dockerfile}
-</CodeBlock>
+The CLI will create a basic `Dockerfile` and a `pipeline.sh` script. We will customise these files to fit our needs as we progress through the workshop. Feel free to explore them.
 
-The Dockerfile above uses the `kratix-pipeline-utility` image as a base image. This image provides a set of utilities that make it easier to write Kratix Pipelines. You may also notice that image will execute a script called `resource-configure` when the container starts.
+The generated Dockerfile uses the `kratix-pipeline-utility` image as a base image. This image provides a set of utilities that make it easier to write Kratix Pipelines. You may also notice that image will execute a script called `pipeline.sh` when the container starts.
 
-Create a file called `resource-configure` in the same directory, and make it executable:
+As we will use this same image to build several stages in the pipeline, let's rename the `pipeline.sh` to `resource-configure` to better reflect its purpose.
 
 ```bash
-touch workflows/resource-configure
-chmod +x workflows/resource-configure
+mv workflows/resource/configure/mypipeline/kratix-workshop-app-pipeline-image/scripts/{pipeline.sh,resource-configure}
 ```
 
-Write the following into the `resource-configure` script:
+Make sure to update the Dockerfile to reflect the new script name. Remove the lines marked in red and add the lines marked in green:
+
+```diff title="app-promise/workflows/resource/configure/mypipeline/kratix-workshop-app-pipeline-image/Dockerfile"
++
+- RUN apk update && apk add --no-cache yq
+- RUN apk update && apk add --no-cache yq kubectl
+
+- ADD scripts/pipeline.sh /usr/bin/pipeline.sh
++ COPY scripts/* /usr/bin/
+
+- RUN chmod +x /usr/bin/pipeline.sh
++ RUN chmod +x /usr/bin/*
+
+- CMD [ "sh", "-c", "pipeline.sh"]
+```
```

Comment:
Image that were generated looks different, for example I don't see `RUN apk update && apk add --no-cache yq kubectl`

Reply:
maybe an outdated version of the cli locally? see https://github.com/syntasso/kratix-cli/blob/main/cmd/templates/workflows/Dockerfile.tpl